workloads/kube-burner/common.sh

```diff
@@ -0,0 +1,117 @@
+
+# Check cluster's health
+if [[ ${CERBERUS_URL} ]]; then
+  response=$(curl ${CERBERUS_URL})
+  if [ "$response" != "True" ]; then
+    echo "Cerberus status is False, Cluster is unhealthy"
+    exit 1
+  fi
+fi
+
+export QPS=${QPS:-10}
+export BURST=${BURST:-10}
+export ES_SERVER=${ES_SERVER:-https://search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com}
+export ES_PORT=${ES_PORT:-443}
+export ES_INDEX=${ES_INDEX:-ripsaw-kube-burner}
+export PROM_URL=${PROM_URL:-https://prometheus-k8s.openshift-monitoring.svc.cluster.local:9091}
+OPERATOR_REPO=https://github.com/cloud-bulldozer/benchmark-operator.git
+export NODE_SELECTOR_KEY="node-role.kubernetes.io/worker"
+export NODE_SELECTOR_VALUE=""
+PROM_TOKEN=$(oc -n openshift-monitoring sa get-token prometheus-k8s)
+export PROM_TOKEN
+export WAIT_WHEN_FINISHED=true
+export WAIT_FOR=[]
+export JOB_TIMEOUT=${JOB_TIMEOUT:-7200}
+export TOLERATIONS="[{key: role, value: workload, effect: NoSchedule}]"
+export WORKLOAD_NODE=${WORKLOAD_NODE}
+export UUID=$(uuidgen)
+export TAINT_NODE=${TAINT_NODE:-0}
+bold=$(tput bold)
+normal=$(tput sgr0)
+
+if [[ ${WORKLOAD_NODE} ]]; then
+  PIN_SERVER=$(oc get node ${WORKLOAD_NODE} -o go-template --template='{{index .metadata.labels "kubernetes.io/hostname" }}')
+  export PIN_SERVER
+fi
+
+log() {
+  echo ${bold}$(date "+%d-%m-%YT%H:%m:%S") ${@}${normal}
+}
+
+taint_node() {
+  log "Tainting node ${1}"
+  oc adm taint node ${1} role=workload:NoSchedule --overwrite
+}
+
+deploy_operator() {
+  log "Cloning benchmark-operator from ${OPERATOR_REPO}"
+  rm -rf benchmark-operator
+  git clone ${OPERATOR_REPO} --depth 1
+  log "Deploying benchmark-operator"
+  oc apply -f benchmark-operator/resources/namespace.yaml
+  oc apply -f benchmark-operator/deploy
+  oc apply -f benchmark-operator/resources/backpack_role.yaml
+  oc apply -f benchmark-operator/resources/kube-burner-role.yml
+  oc apply -f benchmark-operator/resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml
+  oc apply -f benchmark-operator/resources/operator.yaml
+  log "Waiting for benchmark-operator to be available"
+  oc wait --for=condition=available -n my-ripsaw deployment/benchmark-operator --timeout=180s
+}
+
+deploy_workload() {
+  log "Deploying benchmark"
+  envsubst < kube-burner-crd.yaml | oc apply -f -
+}
+
+wait_for_benchmark() {
+  rc=0
+  log "Waiting for benchmark kube-burner-${1}-${UUID} to complete"
```

Comment:
We might want to stream the logs of the kube-burner pods to stdout. It will be very useful in case of jenkins to see the progress. Thoughts?

Reply:
nice, I like it